java/piranha/config/properties.json

```diff
@@ -19,12 +19,12 @@
     },
     {
       "methodName": "putToggleEnabled",
-      "flagType": "empty",
+      "flagType": "set_treated",
       "argumentIndex": 0
     },
     {
       "methodName": "putToggleDisabled",
-      "flagType": "empty",
+      "flagType": "set_control",
```

Comment:
Options here include adding a new type (as here) vs expanding the attribute list. Can you please document the reason behind this choice?

Reply:
`empty` was always a mis-specification for these two methods. They have specific semantics as flag setters within an experimentation API, not merely "this method should be removed/a no-op". It's just that before we didn't have any handling specific to flag setting. I am not sure this needs documentation when `empty` itself was undocumented, but if you have a place in particular where you want this design decision documented beyond this comment, I'd be happy to :)

Reply:
I propose adding a separate design doc in the repo (initially under Java) where we can document our design choices. For instance, the discussion [here](https://github.com/uber/piranha/issues/115#issue-851676528) could be added to this doc when the issue is resolved. This can help us with enhancements for other languages too.

Reply:
As discussed offline, rather than adding a design document tracked with the code, my favored resolution is to:

1. Update `README.md` to mention the new values for `flagType`
2. Update #115 with some notes on why we are adding the new `flagType` (and why that property is misnamed, as it technically represents an **experiment API method type** :) )